src/util/util_log.cpp

```diff
@@ -59,7 +59,12 @@ namespace dxvk::log {
         print(message);
         if (filestream) {
             std::scoped_lock lock(fileStreamMutex);
-            filestream << message << std::endl;
+            filestream
+                << "DXVK-NVAPI|"
+                << ::GetCurrentProcessId() << "|"
+                << ::GetCurrentThreadId() << "|"
```

Comment:
IIRC Wine itself logs IDs using a fixed-width hex format, maybe we should do the same? It would make filtering and correlating with Wine logs easier, especially if both end up in a single Proton log file.

But last I checked doing this the C++ way by manipulating the stream's formatting flags was quite messy.

Reply:
an nice, being close to Wine is indeed a good idea. I've pushed that. So prefix lowercase, pid en tid in hex padded with 0 to 4 positions and back to `:`  as separator.

Reply:
Cool, I think it's better now.

Technically Wine prints timestamp, then pid and tid, then log level (which we don't have so whatever), then component/channel name, then function name. But I'm not sure if we want to be _this_ close because at least the timestamp part could be tricky to replicate.

Keeping `dxvk-nvapi` first makes our messages stand out a bit but in a useful manner so there's nothing wrong with leaving this as-is.

Reply:
I looked very briefly at the wine timestamp, since it is not unix time, but based on machine up time, it would require more work. I can move ofc `dxvk-nvapi` after the pid,tid.